modules/light-clients/08-wasm/types/store_test.go

```diff
@@ -373,21 +384,15 @@ func (suite *TypesTestSuite) TestNewMigrateClientWrappedStore() {
 	}
 }
 
-// GetSubjectAndSubstituteStore returns a subject and substitute store for testing.
+// GetSubjectAndSubstituteStore returns two dummy KVStores for testing the migrate client wrapping store.
```

Comment:
minor nit in the comment, I think we should remove "dummy" since they're actual stores and I find it offensive to my kind. 

```suggestion
// GetSubjectAndSubstituteStore returns two KVStores for testing the migrate client wrapping store.
```